doc/content/documentation/systems/Materials/ConcreteASREigenstrain.md

```diff
@@ -0,0 +1,265 @@
+|       2     | $\sigma_u$ |        $0$ |         $1/2$    |   $0$                 |       $0$         |
+|       3     | $\sigma_u$ | $\sigma_u$ |           $1$    | $1/3$                 |       $0$         |
+|       4     |        $0$ | $\sigma_u$ |         $1/2$    |   $0$                 |       $0$         |
+|       5     |      $f_c$ |        $0$ |         $1/2$    |   $0$                 |       $0$         |
+|       6     |      $f_c$ | $\sigma_u$ |           $1$    | $1/2$                 |       $0$         |
+|       7     |      $f_c$ |      $f_c$ |           $1$    |   $1$                 |     $1/3$         |
+|       8     | $\sigma_u$ |      $f_c$ |           $1$    | $1/2$                 |       $0$         |
+|       9     |        $0$ |      $f_c$ |         $1/2$    |   $0$                 |       $0$         |
+|      10     |      $f_t$ |      $f_c$ |         $1/2$    |   $0$                 |       $0$         |
+|      11     |      $f_t$ | $\sigma_u$ |         $1/2$    |   $0$                 |       $0$         |
+|      12     |      $f_t$ |        $0$ |         $1/3$    |   $0$                 |       $0$         |
+|      13     |      $f_t$ |      $f_t$ |         $1/3$    |   $0$                 |       $0$         |
+|      14     |        $0$ |      $f_t$ |         $1/3$    |   $0$                 |       $0$         |
+|      15     | $\sigma_u$ |      $f_t$ |         $1/2$    |   $0$                 |       $0$         |
+|      16     |      $f_c$ |      $f_t$ |         $1/2$    |   $0$                 |       $0$         |
+
+
+The previous steps are then repeated to calculate the ASR expansion weights along the other two principal directions, $W_l(\sigma_k, \sigma_l, \sigma_m)$ and
+ $W_m(\sigma_k, \sigma_l, \sigma_m)$. Note that the summation of $W_k$, $W_l$ and $W_m$ equals 1.
+The individual incremental ASR strains along the principal directions are then obtained using these relative weights by the following formula:
+\begin{equation}
+\label{eqn:asr_principal}
+\begin{array}{llll}
+    \Delta\varepsilon_{i}^{ASR} = W_i\Delta\varepsilon_V^{ASR}, &i =1, 2&and&3.
+\end{array}
+\end{equation}
+Unlike isotropic ASR swelling models, the incremental ASR strains along principal directions obtained by the above formula are in general
+different from each other, depending on the local stress state driven by the material confinement conditions.
+
+Finally, the full ASR expansion-induced incremental strain tensor $\Delta\varepsilon^{ASR}$ in the Cartesian coordinate system can then be obtained by
+rotating $\Delta\varepsilon_{i}^{ASR}$ into the current coordinates via
+\begin{equation}
+\label{eqn:asr_strain}
+\Delta\varepsilon^{ASR}=R\{\Delta\varepsilon_{i}^{ASR}\}R^T.
+\end{equation}
+in which $\Delta\varepsilon^{ASR}$ is a 'chemically' imposed incremental strain tensor at each quadrature point, which is used by the solid mechanics model in Grizzly to drive the deformation of concrete structures.
```

Comment:
This line refers to both the solid mechanics module and Grizzly, which seem like outdated references.